scripts/check_style.sh

```diff
@@ -2,13 +2,20 @@
 
 . scripts/report_errors.sh
 
+EXCLUDE_FILES=(
+  "libsolutil/picosha2.h"
+  "test/libsolutil/UTF8.cpp"
+)
+EXCLUDE_FILES_JOINED=$(printf "%s\|" "${EXCLUDE_FILES[@]}")
+EXCLUDE_FILES_JOINED=${EXCLUDE_FILES_JOINED%??}
+
 (
 REPO_ROOT="$(dirname "$0")"/..
 cd $REPO_ROOT
 
 WHITESPACE=$(git grep -n -I -E "^.*[[:space:]]+$" |
   grep -v "test/libsolidity/ASTJSON\|test/libsolidity/ASTRecoveryTests\|test/compilationTests/zeppelin/LICENSE" |
-  grep -v -E "test/libsolidity/syntaxTests/comments/unicode_direction_override_1.sol"
```

Comment:
What was the reason for this?

Reply:
I think this was an attempt at fixing something that in the end was fixed differently.